tlaplus/README.md

```diff
@@ -0,0 +1,7 @@
+This folder contains TLA+ proofs for various subsystems of the project including:
+
+* Distributed Dynamodb lock
+* S3 multi-writer design
+
+To model check these specs, you need to use [TLA+
+toolbox](https://lamport.azurewebsites.net/tla/toolbox.html).
```

Comment:
Might be worth mentioning [this confusing behavior](https://github.com/tlaplus/tlaplus/issues/350) when opening a spec in the toolbox in the README.